snapshots/output/multi-project/packages/a/src/index.ts

```diff
@@ -1,6 +1,6 @@
   export function a(): string {
 //                ^ definition @example/a 1.0.0 src/`index.ts`/a().
-//                documentation ```ts\n() => string\n```
+//                documentation ```ts\nfunction a() => string\n```
```

Comment:
I would expect the syntax to use `:` as a separator for functions
```suggestion
//                documentation ```ts\nfunction a(): string\n```
```